File: frontend/src/app/models/product-to-insert.ts
export class ProductToInsert {
    id: number
    image: File | null
    name: string
    description: string
    price: number
    stock: number
    categoryName: string

    constructor(image: File | null, name: string, description: string, price: number, stock: number, categoryName: string, id: number)
    {
        this.image = image
        this.name = name
        this.description = description
        this.price = price
        this.stock = stock,
        this.categoryName = categoryName
        this.id = id
    }
}

File: backend/Server/Server/DTOs/ProductToInsert.cs
using Server.Models;

namespace Server.DTOs;

public class ProductToInsert
{
    public string Id { get; set; }
    public string Name { get; set; }

    public string Description { get; set; }

    public string Price { get; set; }

    public string Stock { get; set; }

    public IFormFile Image { get; set; }

    public string CategoryName { get; set; }

}
